lib/build.sh

```diff
@@ -20,3 +20,10 @@ check_tsc_binary() {
 
   [[ -f "$build_dir/node_modules/typescript/bin/tsc" ]]
 }
+
+check_tsc() {
+  local build_dir=$1
+  if ! detect_out_dir "$build_dir" ; then
+    tsc
+  fi
+}
```

Comment:
Can we add new lines to the files? 😃 While it probably doesn't matter when we're using bash, it's usually good process since some programs process end of files with a new line. This SO has a good discussion: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline